tests/topotests/kernel_bypass_route_map_zebra/test_kernel_bypass_ecmp.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/python
+# SPDX-License-Identifier: ISC
+#
+# test_kernel_bypass_ecmp.py
+# Part of NetDEF Topology Tests
+#
+# Copyright (c) 2017 by
+# Network Device Education Foundation, Inc. ("NetDEF")
+#
+
+"""
+test_kernel_bypass_ecmp.py: Test kernel bypass route-map.
+"""
+from lib.topolog import logger
+from lib.topogen import Topogen, TopoRouter, get_topogen
+from lib import topotest
+import os
+import sys
+import pytest
+import json
+from functools import partial
+# Save the Current Working Directory to find configuration files.
+CWD = os.path.dirname(os.path.realpath(__file__))
+sys.path.append(os.path.join(CWD, '../'))
+
+# pylint: disable=C0413
+
+
+def build_topo(tgen):
+    r1 = tgen.add_router("r1")
+    r2 = tgen.add_router("r2")
+
+    switch = tgen.add_switch('s1')
+
+    switch.add_link(tgen.gears['r1'])
+    switch.add_link(tgen.gears['r2'])
+
+    r3 = tgen.add_router("r3")
+    switch = tgen.add_switch('s2')
+    switch.add_link(tgen.gears['r1'])
+    switch.add_link(tgen.gears['r3'])
+
+
+def setup_module(mod):
+    "Sets up the pytest environment"
+    # This function initiates the topology build with Topogen...
+    tgen = Topogen(build_topo, mod.__name__)
+    # ... and here it calls Mininet initialization functions.
+    tgen.start_topology()
+    print("topology started")
+    # This is a sample of configuration loading.
+    router_list = tgen.routers()
+
+    # For all registred routers, load the zebra configuration file
+    for rname, router in router_list.items():
+        if rname == "r1":
+            router.load_frr_config(
+                os.path.join(CWD, "{}/frr_routemap.conf".format(rname))
```

Comment:
Why do we need two files frr.conf and frr_routemap.conf for r1?

Reply:
I want to use 2 frr.conf for 2 test "test_ecmp", "test_reconfig" before. 
"test_ecmp" is to test route-map is configured before zebra init and "test_ecmp" is to test route-map is configured after zebra init.
I find  "test_reconfig" can cover "test ecmp" normal case now, I will remove  "test_ecmp" now.